swingset/src/main/java/com/nqadmin/swingset/SSDBComboBox.java

```diff
@@ -196,7 +196,7 @@ private static class GlazedModel extends SSBaseComboBox.BaseGlazedModel<Long, Ob
 	 */
 	@Override
 	protected SSListItem createNullItem(Model.Remodel remodel) {
-		return remodel.createOptionMappingItem(null, "", null);
+		return remodel.createOptionMappingItem(null, null, null);
```

Comment:
Looks good. Nulls and Dates, who would've thought...

BTW, prompted me to take another look at `SSListItemFormat.appendValue(). Looks like `SSListItemFormat.setPattern()` is _only meaningful for date-time_ types. Sigh, now I'll obsess about it. One idea is that `setPattern()` should be deprecated in favor of `setFormatAndPattern()` then it could be readily used to format any jdbc data type.

Any objections if I take a look at that? If toString() is good enough for now and there's no short term need, then maybe I should do that to the formatting and masks branch.